WhatsNew.Cli/README.md

```diff
@@ -0,0 +1,131 @@
+﻿## Onboard a repository / docset
+
+### Create a configuration file
+
+Complete the following steps in Visual Studio Code:
+
+1. Create a JSON configuration file within the GitHub repository:
+    - For docsets in the *azure-docs-pr* repository, the file's naming convention is *.\{SERVICE-NAME}.json*. The file should live in the repository's *.whatsnew* directory. Replace the *\{SERVICE-NAME}* placeholder with the directory name within *articles* in which the docs are stored. For example, name the file *.app-service.json* for Azure App Service docs.
+    - For all other docsets, the file should be named *.whatsnew.json* and live in the repository's root directory.
+1. Associate your JSON configuration file with the JSON schema by adding a `$schema` property to line 2 of the JSON configuration file. The `$schema` property's value should be set as follows:
+
+    ```json
+    "$schema": "https://whatsnewapi.azurewebsites.net/schema",
```

Comment:
BTW, this API is no longer available. It was an ASP.NET Core web API running in App Service that provided the JSON schema. You'd need this `$schema` line to get IntelliSense in your JSON file. Let me know if you have an Azure subscription we can use, and I can get this going again.

Reply:
Would it be better to put the schema here in this repo? It's a public URL.

Reply:
That could work.